stores/types.go

```diff
@@ -15,6 +17,7 @@ import (
 var zeroCurrency = currency(types.ZeroCurrency)
 
 type (
+	datetime       time.Time
```

Comment:
<https://github.com/mattn/go-sqlite3/issues/951>
<https://github.com/mattn/go-sqlite3/issues/748>
ermagherd

Reply:
Alternatively you can copy what we do for hosts and use `int64` and a unix timestamp.

Reply:
decided to (re)use `created_at` for the time being and save some space, we can also add a `mod_time` column that's an `int64` later